CHANGELOG.md

```diff
@@ -1,8 +1,18 @@
+# 1.32.0
```

Comment:
FYI - as a drive by comment - it looks like `1.31.0` has not been published; the latest published version is `1.30.0`.

Reply:
@pq has been experimenting not publishing linter, as there is no known use case for publishing it.

Reply:
OK, that makes a certain amount if sense :)

Perhaps change the listing on pub.dev to 'unlisted'? That'll mean it won't show up in search results.

If we know were never going to publish it again (which it sounds like we're not yet certain of?) it can instead be marked 'discontinued' in pub.

Reply:
Unlisted is an interesting option.  We've got a few dependent packages so we'll want to sort that out before we go too far.  Maybe we can discuss in person?